demo/tests/test_conditions.py

```diff
@@ -87,6 +87,30 @@ def test_jedi_displayed(self):
         self.assertFalse(form.is_valid())
         self.assertIn('bar', form.errors)
 
+    def test_jedi_displayed_without_condition_name(self):
+        self.formidable.conditions = [
+            {
```

Comment:
What's going on if you add two or more rules without names, is it ok?

Reply:
Yep, I'm going to write some additional tests for better coverage